devsetup/scripts/gen-ansibleee-ssh-key.sh

```diff
@@ -57,3 +59,4 @@ EOF
 oc create -f ansibleee-ssh-key-secret.yaml
 
 popd
+popd
```

Comment:
@slagle nested, popd's? Maybe we should get rid of pushd/popd and just use a clean 'cd' to whatever directory approach instead